src/Files.Core/ViewModels/Dialogs/ReleaseNotesDialogViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Core.ViewModels.Dialogs
+{
+	public sealed class ReleaseNotesDialogViewModel : ObservableObject
+	{
+		public string _ReleaseNotesMadrkdown = string.Empty;
```

Comment:
```suggestion
		private string _ReleaseNotesMadrkdown;
```
It should be private and specifying an initial value is meaningless because the constructor always overwrites the value.